battleship/board.py

```diff
@@ -1,13 +1,28 @@
 DIRECTIONS = ('NS', 'EW')
 OCEAN = 'O'
 
+
 class Board(object):
     def __init__(self):
         self.board = [[OCEAN for _ in range(10)] for _ in range(10)]
+        self.hit_ships = []
+        self.attacks = []
```

Comment:
I don't think we need to hold this information. Instead, let's make each attack site an M for Miss or an H for Hit. That way a player can print their current board at any point in the game.